docs/helm/README.md

```diff
@@ -273,6 +273,20 @@ at least 3 nodes with 12 cpu cores and 32 GB of memory in total. You can install
 requirements by fine tuning the [resource requests](https://kubernetes.io/docs/concepts/configuration/manage-resources-containers/#resource-units-in-kubernetes)
 available in the [values](https://github.com/Alfresco/acs-deployment/blob/master/helm/alfresco-content-services/values.yaml) for each component.
 
+To install older versions get the specific values from [this folder](https://github.com/Alfresco/acs-deployment/tree/master/helm/alfresco-content-services).
```

Comment:
consider adding an heading so it's more easily searchable and could also be linked from the detailed guides.

just an idea but ideally all the detailed deployment guides we have should at the end link at this document for the general information around helm